src/common/latest/config.ts

```diff
@@ -1,42 +1,42 @@
 // When the user does not specify a specific version, these willl be the default versions used.
-const msBuildVersion = "6.2.0.85879";
-const cliVersion = "6.1.0.4477";
+const dotnetScannerVersion = "6.2.0.85879";
+const cliScannerVersion = "6.1.0.4477";
 
 // MSBUILD scanner location
 const dotnetScannersBaseUrl = `https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/`;
 
 // CLI scanner location
 const cliScannerBaseUrl = "https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/";
 
-function getMsBuildClassicFilename(msBuildVersion: string) {
-  return `sonar-scanner-${msBuildVersion}-net-framework.zip`;
+function getMsBuildClassicFilename(dotnetScannerVersion: string) {
+  return `sonar-scanner-${dotnetScannerVersion}-net-framework.zip`;
 }
 
-function getMsBuildDotnetFilename(msBuildVersion: string) {
-  return `sonar-scanner-${msBuildVersion}-net.zip`;
+function getMsBuildDotnetFilename(dotnetScannerVersion: string) {
+  return `sonar-scanner-${dotnetScannerVersion}-net.zip`;
 }
 
-function getCliScannerFilename(cliVersion: string) {
-  return `sonar-scanner-cli-${cliVersion}.zip`;
+function getCliScannerFilename(cliScannerVersion: string) {
+  return `sonar-scanner-cli-${cliScannerVersion}.zip`;
 }
 
-function msBuildUrlTemplate(msBuildVersion: string, framework: boolean) {
+function msBuildUrlTemplate(dotnetScannerVersion: string, framework: boolean) {
```

Comment:
shouldn't we rename the method call too to align? 
```suggestion
function dotnetUrlTemplate(dotnetScannerVersion: string, framework: boolean) {
```